tests/framework/PostProcessors/Validation/test_validation_gate_representativityLinModel.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <RunInfo>
+    <WorkingDir>Representativity</WorkingDir>
+    <Sequence>mcRun, PP1</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/Validation/test_validation_gate_representativity</name>
+    <author>Mohammad Abdo (@Jimmy-INL)</author>
+    <created>2021-04-29</created>
+    <classesTested>PostProcessors.Validation</classesTested>
+    <description>
+        This test is aimed to show how to use the mechanics of the Validation Post-Processor. For semplicity,
+        this test is using the attenuation model (analytical) and simple representativity factors metrics.
+        The output name convention is ``feature name''\underscore``target name''\underscore``metric name''=.
```

Comment:
I think you need to update the descriptions for the tests that you added to reflect the problems. (comments valid for other test files also)

Reply:
Done